packages/gen-ai/frontend/src/app/Chatbot/ChatbotPlayground.tsx

```
@@ -281,7 +269,8 @@ const ChatbotPlayground: React.FC<ChatbotPlaygroundProps> = ({
           setIsNewChatModalOpen(false);
         }}
         onConfirm={() => {
-          chatbotMessages.clearConversation();
+          // Clear all chatbot instances
+          messageHooksRef.current.forEach((hook) => hook.clearConversation());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)